FrontEnd/src/components/adminPage/menu/Menu.jsx

```diff
@@ -10,19 +13,29 @@ const MENU = [
     {
         id: 'am2',
         title: 'Компанії',
-        link: '/customadmin/companies/'
+        link: '/customadmin/profiles/'
     },
 ];
 
 function Menu() {
+    const auth = useAuth();
+    const handleLogout = async () => {
+        if (auth.isAuth) {
+            axios.post(`${process.env.REACT_APP_BASE_API_URL}/api/auth/token/logout`)
+                .then(() => {
+                    auth.logout();
+                });
+        }
+    };
```

Comment:
We may use `logout` method from the `useAuth`

Reply:
Done